mars/services/task/execution/ray/config.py

```diff
@@ -18,10 +18,22 @@
 from ..utils import get_band_resources_from_config
 
 
+# the default times to retry subtask.
+DEFAULT_SUBTASK_MAX_RETRIES = 3
+
+
 @register_config_cls
 class RayExecutionConfig(ExecutionConfig):
     name = "ray"
 
+    def __init__(self, execution_config: Dict):
+        super().__init__(execution_config)
+        self._subtask_max_retries = (
+            self.get_execution_config()
```

Comment:
We can access the config dict by `self._execution_config` instead of public method `self.get_execution_config()`.